WordPress/Classes/ViewRelated/Jetpack/Jetpack Restore/Restore Complete/JetpackBackupCompleteViewController.swift

```diff
@@ -49,7 +49,12 @@ class JetpackBackupCompleteViewController: BaseRestoreCompleteViewController {
     // MARK: - Private
 
     private func downloadFile() {
-        // TODO
+        guard let url = backup.url,
+              let downloadURL = URL(string: url) else {
+            return
```

Comment:
I wonder if in the future we should show an alert letting the user know their download URL didn't work. Thoughts?

Reply:
Fixed! 503b56a